src/index.js

```diff
@@ -439,53 +438,44 @@ async function fetchHousingInfos() {
     log('info', 'No properties owned, returning null')
     return null
   }
-  const realEstatePageUnspaced = realEstatePage.replace(/[ ]{2,}/g, '')
```

Comment:
Can you add a big try catch around the housing scraping
From L426 to the end, and throw a warning in a housing info failed.

This way, a housing scraping will let the identity to be saved.